### PR TITLE
Added finnish translation

### PR DESCRIPTION
**Pull request by pasiz**
Finnish translation for the vacuum card

### CODE REVIEW

**Review by denysdovhan — APPROVED** [review]
Thanks for contributing!

**Review by denysdovhan — CHANGES_REQUESTED** [review]
Wait a minute! 
You should add the localization file in `localize.js` and add the corresponding line in docs!

**Comment by pasiz** [comment]
Sorry, i forgot to commit those files on git

File: src/localize.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ import * as sv from './translations/sv.json';
 import * as nb from './translations/nb.json';
 import * as da from './translations/da.json';
 import * as ko from './translations/ko.json';
+import * as ko from './translations/fi.json';
```
Comment (denysdovhan) [inline src/localize.js:20]: `ko` is already declared above. This will cause a syntax error.

**Review by denysdovhan — CHANGES_REQUESTED** [review]
There's no `fi.json` in this PR. Please, test your PR locally before opening a PR 🙏🏻

**Review by denysdovhan — APPROVED** [review]

**Comment by denysdovhan** [comment]
Thanks for contributing!

**Pull request merged**